user.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+users = {}
+regex_member = "[a-zA-Z0-9- .]+$"
+regEmail = r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)"
+regPassword = r"^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$"
+
+class Users(object):
+    """
+      Users Class to handle  user functions like login and signup
+      """
+    def __init__(self, member=None, email=None, password=None):
+        """ Initializing  class instance variables"""
+        self.member = member
+        self.email = email
+        self.password = password
+
+    def member_register(self, email, member, password, cpassasword):
+        """ method that defines the elements required to create an account """
+        if re.match(regex_member, member):
+            if member != '' and email != '' and password != '' and cpassasword != '' and cpassasword != ' ':
+                if member not in users.keys():
+                    if email not in users.keys():
+                        if password == cpassasword:
+                            if re.search(regEmail, email):
+                                if re.search(regPassword, password):
+                                    users[email] = {'member': member,'email': email,'password': password,}
+                                    return "200,OK"
+                                return "205,Password Regex mismatch"
+                            return "205,Email Regex mismatch"
+                        return "400,Passwords dont match"
+                    return "401,Email exists"
+                return "401,Member exists"
+            return "205,Invalid input"
+        return "205,Regex mismatch"
+
+    def user_login(self, email, password):
+        """Method to login user"""
+        if email != ' ' and password != ' ':
```

Comment:
`!= ' '`. This seems to occur everywhere in your code. Use `if not x: statement`